src/api/java/appeng/api/networking/crafting/ICraftingCallback.java

```diff
@@ -24,6 +24,9 @@
 package appeng.api.networking.crafting;
 
 
+import javax.annotation.Nonnull;
+
+
 public interface ICraftingCallback
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "ICraftingCallback" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)